authentification/transport/http/login.go

```diff
@@ -0,0 +1,35 @@
+package http
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"log"
+
+	"github.com/There-is-Go-alternative/GoMicroServices/authentification/usecase"
+	"github.com/gofiber/fiber"
+)
+
+func LoginHandler(cmd usecase.LoginProto) fiber.Handler {
+	return func(c *fiber.Ctx) {
+		var dto usecase.LoginDTO
+
+		ctx, cancel := context.WithTimeout(context.Background(), 100*time.Second)
+		err := c.BodyParser(&dto)
+		defer cancel()
+		if err != nil {
+			log.Printf("LoginInput invalid: %v\n", dto)
+			c.Status(http.StatusNotFound).JSON(err)
```

Comment:
Si l'utilisateur n'a pas rentré les bonnes informations pour se connecter, on a tendance à mettre un status 400 (bad request). Par ailleurs, dans les autres services, nous avons fais un système de réponse sous la forme:
`
{
"success": true/false,
"data/message": "message d'erreur custom"
}
`
ça permet d'avoir une norme et qu'on se comprenne tous :)
```suggestion
			c.Status(http.StatusBadRequest).JSON(err)
```